src/vtlengine/Exceptions/messages.py

```diff
@@ -181,6 +181,16 @@
     # flow_to_stock, stock_to_flow
     "1-1-19-8": "At op {op}: {op} can only be applied to a {comp_type}",
     "1-1-19-9": "At op {op}: {op} can only be applied to a {comp_type} with a {param}",
+    #Other time operators
```

Comment:
All new messages defined here should start with 2- as they refer to values (RuntimeErrors)